### ISSUE TITLE
Backend refactoring

### ISSUE THREAD

**Issue by Idontker** [amosproj/amos2022ss04-digital-identity#121]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Code is cleaned up
* Architecture is updated to the expected future needs
* Interfaces are updated and in documentation

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
- [ ] QR-Code als Service
- [ ] controller, services, model beibehalten 
- [ ] Controller Funktionalität auf “forwarding” beschränken 
  - Restliche Funktionalitäten im Service
  - Model: nur Datenbank
- [ ] messages/answers und messages/responses. Einsortieren
  - [ ] CreateConnection
  - [ ] Invitation 
- [ ] Autoformat:
  - [ ] CreateConnection
  - [ ] Invitation
- [ ] Authentification:
  - [ ] Controller
    - [ ] register, update raus
    - [ ] nur /login und passwort setzten geschichten 
  - [ ] Service:
    - [ ] eigentliche Überprüfung
    - [ ] + verifyAuthentification() für andere Controller zur Verfügung stellen
  - [ ] => ggf. nicht notwendig, weil MAGIIIIEEEE
  - [ ] User spezifischer String (in User.java und ConnectionController.java) in messages/answers auslagern
  - [ ] User:
    - [ ] remove birthday 
    - [ ] remove company
    - [ ] remove team
  - [ ] LissiApi: baseURL auslagern
